carp.deployments.core/src/commonMain/kotlin/dk/cachet/carp/deployments/application/StudyDeploymentStatus.kt

```diff
@@ -50,6 +57,7 @@ sealed class StudyDeploymentStatus
      */
     @Serializable
     data class DeploymentReady(
+        override val createdOn: Instant,
         override val studyDeploymentId: UUID,
         override val devicesStatus: List<DeviceDeploymentStatus>,
         override val startedOn: Instant?
```

Comment:
`readyOn`? I guess it marks the time when the client signal the fetched deployment is 'ready'. It appears the use of 'started' was deliberately avoided here in the first place since we cannot know whether the client is actually running the study.

Reply:
No, this is the time the deployment was created, coinciding with when invitations were sent out. `startedOn` is I think what you mean by `readyOn`. Maybe a good candidate for a rename 🤔 .

`startedOn` is set once all devices participating in the deployment have called `deploymentSuccessful`.

Reply:
Oh, I now totally see you did comment on the correct line. I got confused because `createdOn` was highlighted as the change. :)